src/scripts/main.js

```diff
@@ -2,7 +2,19 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const header = document.querySelector('.dashboard').rows[0];
+const newRow = document.createElement('tr');
 
-// write your code here
+for (let i = 0; i < people.length; i++) {
+  newRow.innerHTML = `
```

Comment:
let's use forEach method,
also instead innerHtml use `insertAdjacentHTML` 
on table
something like that
```
people.forEach(({ name: personName, sex, born, died }) => {
  table.insertAdjacentHTML('beforeend', `
    <tr>
      <td>${personName}</td>
      <td>${sex === 'm' ? 'Male' : 'Female'}</td>
      <td>${born}</td>
      <td>${died}</td>
      <td>${died - born}</td>
      <td>${Math.ceil(died / 100)}</td>
    </tr>
  `);
});
```